src/app/shared/dropdown/dropdown.css

```diff
@@ -36,7 +36,7 @@
   overflow-y: hidden;
   position: absolute;
   right: 0;
-  z-index: 999999;/* zendesk widget + 1 */
+  z-index: 997;
```

Comment:
was looking at z-index for dropdowns vs the modal overlay and realized this sucker should only apply to dropdowns in the nav bar area.